WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/list/OrderListFragment.kt

```
@@ -709,7 +709,9 @@ class OrderListFragment :
     }
 
     private fun showOrderFilters() {
-        findNavController().navigateSafely(R.id.action_orderListFragment_to_orderFilterListFragment)
+        findNavController().navigateSafely(
```

This function catches the exception internally, preventing the crash. The crash is happening because the Filters screen is already visible, so that's why I decided to swallow the exception.

---

The code change seems fine since it can't be reproduced. 

Strange that is started occurring 6 months ago
in release 17.7-rc-2. I don't see which change might have caused that.